documentation/source/user-manual/configuration.rst

```diff
@@ -0,0 +1,56 @@
+Styles
+======
+
+Styles are defined in a configuration file by :py:class:`stylist.style.Style`
+objects.
+
+Extending an Existing  Style
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To add a new rule to an existing style just add an instance of the rule class
+to the constructor argument list. You may need to modify the ``import``
+statements to make it available for use.
+
+For instance:
+
+.. code-block:: diff
+
+   from stylist.fortran import (FortranCharacterset,
+  -                             MissingImplicit)
+  +                             MissingImplicit,
+  +                             MissingOnly)
+   from stylist.rule import TrailingWhitespace
+   from stylist.style import Style
+
+   lfric = Style(
+                 FortranCharacterset(),
+                 TrailingWhitespace(),
+  -              MissingImplicit()
+  +              MissingImplicit(),
+  +              MissingOnly(ignore=['pfunit_mod', 'xios'])
+                )
+
+Adding a New Style
+~~~~~~~~~~~~~~~~~~
+
+As intimated above, this a simple matter of creating a new style object
+variable.
+
+.. code-block:: diff
+
+   from stylist.fortran import (FortranCharacterset,
+                                MissingImplicit,
+                                MissingOnly)
+   from stylist.rule import TrailingWhitespace
+   from stylist.style import Style
+
+   lfric = Style(
+                 FortranCharacterset(),
+                 TrailingWhitespace(),
+                 MissingImplicit(),
+                 MissingOnly(ignore=['pfunit_mod', 'xios'])
+                )
+
+  +special = Style(FortranCharacterset(),
+  +                TrailingWhitespace(),
+  +                MissingImplicit())
```

Comment:
Should this "special" style be formatted the same as the lfric one above? i.e. a new line after the first `(` and before the last `)`? (I'm probably being overly pedantic so won't formally request a change, but go ahead if it bugs you 😀)